### DISCUSSION TITLE
Can you help this non-expert with settings in Asreview 2.0 for Phase 3 of Safe procedure?

### DISCUSSION THREAD

**Discussion by Annasmits** [asreview/asreview#2208] [Q&A] [answered]
Hi there, 

I am working on a large-scale scoping review (mostly qualitative data) and have finished the first round of screening (abstracts). I am following the SAFE procedure and have finished phase 2. Now I want to continue with phase 3. With the old version of ASreview, I determined that I should export the labeleled dataset > import it and then change the model to SBert & Neural net. However, now I've upgraded to Asreview 2.0. Which gives me the opportunity to continue with a different model (NICE!), but does not contain the options Sbert & Neural Net. 
Can someone help me with what I should select for phase 3? Something that fits with this passage from the SAFE procedure paper:

This problem is best tackled with deep
learning models, which are better at finding complex
connections within data than shallow networks like the
simple model used in the first screening phase. However,
deep learning models require more training data [2] and
are not expected to perform well in the first few iterations
[39]. Hence, the labeling decisions from the first
screening phase will be used as prior knowledge to train
a different model, for example a neural network model as
a classifier along with sBert as the feature extractor.

Thanks a lot!

**Comment by timovdk** [comment, marked as answer]
Hi @Annasmits, 
With ASReview we have also released [ASReview-Dory](https://github.com/asreview/asreview-dory). This will add support for the deep learning models that the SAFE procedure talks about. You can install it using `pip install asreview-dory`. If you have used deep learning models in ASReview 1, please also make sure to uninstall Tensorflow: `pip uninstall tensorflow`.

With ASReview-Dory installed, you have the following options, both would suit the SAFE procedure, but only one option specifically uses a neural network as the classifier. I would personally choose ELAS-h3 (option 1) because the feature extractor does a lot of the heavy lifting anyways. However, if you also want to use a deep learning classifier, choose option 2. 

Please note that the feature extractor can take a while to vectorize all your data (depending on hardware and the size of your dataset). You can either switch the model, and keep screening, and at some point the new model is ready. However, you could also label 1 record after switching the model (this triggers ASReview to start preparing the new model), take a break, and come back after some hours.

## 1. Select `ELAS-h3` (MXBAI sentence embeddings and SVM classifier)
![Screenshot 2025-05-16 at 10 49 43](https://github.com/user-attachments/assets/6af1a733-0d4a-4a85-8467-099bd4e1b061)

## 2. Select `Custom` with the following options:
![Screenshot 2025-05-16 at 10 48 54](https://github.com/user-attachments/assets/19e017be-49a7-4a30-8103-f455835de55d)

**Comment by Annasmits** [comment]
Thank you so much Timo! Very helpful
